web/src/scroll_util.ts

```diff
@@ -1,7 +1,10 @@
 import $ from "jquery";
 import SimpleBar from "simplebar";
 
-export function get_content_element($element) {
+// This type is helpful for testing, where we may have a dummy object instead of an actual jquery object.
+type JQueryOrZJQuery = {__zjquery?: true} & JQuery;
```

Comment:
I was just wondering if it's really necessary to create this type just for testing purposes. Do we actually use types in our tests?

Also, I'm a bit concerned that if we want to add a new test, we'll need to make changes to the actual code and modify the type from JQuery to JQueryOrZJQuery. That seems like it could be a bit of a hassle.

What do you think?